assets/combined.css

```diff
@@ -680,25 +702,28 @@ figure blockquote {
   padding: 26px 0px;
   padding: 1.625rem 0rem;
 }
+
 @media screen and (min-width: 40em) {
   figure blockquote {
     padding: 62px 0px 31px;
-    padding: 3.444444444444445rem 0rem 1.722222222222222rem;
```

Comment:
I know, those decimals felt like a cheat fix to me in the first place tbh. Ideally we wouldn't have to rely on those but I'm not sure how that would be possible... I spent quite some time on this and this was the only way to make the grid and the elements fit. I'll update the br margin and the other bug with indenting and let's see where we go from there...